config/config_test.go

```diff
@@ -118,8 +118,10 @@ func TestGetDestinationRPCEndpoint(t *testing.T) {
 	}
 
 	for i, testCase := range testCases {
-		res := testCase.s.GetNodeRPCEndpoint()
-		require.Equal(t, testCase.expectedResult, res, fmt.Sprintf("test case %d failed", i))
+		t.Run(fmt.Sprintf("test_%d", i), func(t *testing.T) {
+			res := testCase.s.GetNodeRPCEndpoint()
+			require.Equal(t, testCase.expectedResult, res)
+		})
```

Comment:
I do like this format a lot more. If we can add names to the tests describing what they do, that would be even better than a number.